source/auth/auth.rst

```diff
@@ -250,6 +250,48 @@ or "SCRAM-SHA-256").
 The cache entry value MUST be either the ``saltedPassword`` parameter or the
 combination of the ``clientKey`` and ``serverKey`` parameters.
 
+Speculative Authentication via Handshake
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:since: 4.4
+
+The ``isMaster`` handshake supports a new argument, ``speculativeSaslStart``,
+provided as a BSON object. Clients specifying this argument to ``isMaster`` will
+speculatively include the first ``saslStart`` command of an authentication handshake.
+This command may be provided to the server in parallel with any standard request for
+supported authentication mechanisms. This would permit clients to merge their
```

Comment:
Does "any standard request for supported authentication mechanisms" refer to anything else besides [mechanism negotiation via handshake](https://github.com/mongodb/specifications/blob/master/source/auth/auth.rst#mechanism-negotiation-via-handshake)? If not, I'd propose adding "(i.e. `saslSupportedMechs`)" or a reference to that section at the very end of this sentence for clarity.

Reply:
I have added your proposed text.